packages/mailer/src/Exceptions/ExpeditorWasMissing.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Tempest\Mail\Exceptions;
+
+use Exception;
+
+final class ExpeditorWasMissing extends Exception implements MailerException
+{
+    public function __construct()
+    {
+        parent::__construct('No expeditor address provided.');
```

Comment:
I don't know what expeditor means? Maybe the exception could elaborate? Maybe I'm overthinking though